Library/Homebrew/test/test_dependency.rb

```diff
@@ -39,6 +39,13 @@ def test_accepts_symbol_and_string_tags
     assert_equal [:build, "bar"], dep.tags
   end
 
+  def test_merge_repeats
```

Comment:
I'd call it `test_merge_repeats_option_names` as it is only testing this aspect of `merge_repeats`.